tests/test_data/alittlebityummy.com/alittlebityummy_1.json

```diff
@@ -0,0 +1,59 @@
+{
+  "author": "Alana Scott",
+  "canonical_url": "https://alittlebityummy.com/recipe/en-nz/low-fodmap-creamy-ham-and-potato-soup-4/",
+  "host": "alittlebityummy.com",
+  "description": "Soup is the BEST. There is nothing like curling up on the couch with super cosy bowl of soup. This low FODMAP creamy ham and potato soup is the perfect option.",
+  "image": "https://alittlebityummy.com/wp-content/uploads/2021/12/Low-FODMAP-Creamy-Ham-Potato-Soup.jpg",
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "195 shaved ham (we used leftover Christmas ham)*",
+        "80 leek (green leaves only, finely chopped)*",
+        "800 potato (peeled & chopped in small chunks)",
```

Comment:
800 potato!?

I think we need to check whether we can gather the measurement units for the ingredients :)

Reply:
That's a whole lot of potatos, should be resolved now!

Reply:
:)  thanks!